weatherman.py

```diff
@@ -1,19 +1,61 @@
+import os
+import glob
 import sys
-sys.argv
 
+sys.argv
 for w in sys.argv:
     print(w)
+if len(sys.argv) < 3:
+    sys.exit(1)
 
 my_weatherlist = []
 
-def func_access(file_name):
-    file = open (file_name , "r+")
-    read = file.read().splitlines()
+def read_files(file_name):
+    file = open (file_name , "r")
+    read = file.read().splitlines(",")
     for i in range(2, len(read)-1):
-        my_weatherlist.append(read[i].split())
-        print(read[i].split())
-    print(read[0])
+        my_weatherlist.append(read[i].split(","))
+        read[i].split(",")
+
+directory = sys.argv[1]
```

Comment:
We are not taking directory name as a command line argument so the directory name will remain the same. While year is given in the command line argument so we have to take care of it.